prod.json

```
@@ -17,7 +17,7 @@
   },
   "popups": {
     "midnightDistribution": {
-      "display": false
+      "display": true
     },
     "generalFeaturesAnnouncement":{
       "display": true
```

```suggestion
      "display": false
```
It should be changed to true to avoid overlapping